### ISSUE TITLE
now() doesn't work

### ISSUE THREAD

**Issue by zrzka** [balena-io-modules/balena-temen#37]
[Not implemented for the wasm32-unknown-unknown target](https://github.com/rust-lang-deprecated/time/blob/master/src/sys.rs#L101).

**Comment by zrzka** [comment]
There's [PR](https://github.com/chronotope/chrono/pull/287) open with initial wasm32 arch support. Should _self fix_ once approved & new chrono released.

**Comment by zrzka** [comment]
Works with ...

```
chrono = { git = "https://github.com/jjpe/chrono", branch = "wasm" }
```

... which is the mentioned PR branch. Waiting for the new chrono release.

**Comment by zrzka** [comment]
Fixed in #39
